### ISSUE TITLE
Build script fails when executing

### ISSUE THREAD

**Issue by daniel-iobuilders** [hyperledger/besu-native#37]
I tried to build the project by executing `build.sh`, but I get two errors when I do it. I executed the following commands:

```
sudo apt-get install build-essential automake autoconf libtool
git submodule update
./build.sh
```

I always get the following errors:
```
./build.sh: line 63: ./autogen.sh: No such file or directory
./build.sh: line 78: cargo: command not found
```

So it seems that `autogen.sh` is missing or not generated. The second one seems related to Rust. I don't have it installed, so I guess it should be added to the README to list which packages need to be installed.

OS: Ubuntu 20.04.2 LTS

**Comment by daniel-iobuilders** [comment]
I opened PR #38 to fix the two errors. But now I get another error:

```
error: failed to select a version for `parity-crypto`.
    ... required by package `eth_pairings v0.6.0 (/home/daniel/projects/besu-native/bls12-381/matterlabs-eip1962)`
versions that meet the requirements `^0.4` are: 0.4.2, 0.4.0

the package `eth_pairings` depends on `parity-crypto`, with features: `publickey` but `parity-crypto` does not have these features.


failed to select a version for `parity-crypto` which could resolve this conflict
```

I have no experience with Rust. Is this a setup problem as well, or is this an unrelated error?

**Comment by daniel-iobuilders** [comment]
Resolved by #40
